VideoLocker/src/main/java/org/edx/mobile/view/CourseDiscussionResponsesFragment.java

```diff
@@ -73,14 +75,31 @@ public void onViewCreated(View view, Bundle savedInstanceState) {
 
         getCommentList(true);
 
-        addResponseTextView.setText(R.string.discussion_responses_add_response_button);
-
-        addResponseLayout.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                router.showCourseDiscussionAddResponse(getActivity(), discussionThread);
-            }
-        });
+        if (discussionThread.isClosed()) {
+            addResponseTextView.setText(R.string.discussion_add_response_disabled_title);
+            addResponseTextView.setCompoundDrawables(
```

Comment:
You can use the `TextViewCompat.setCompoundDrawablesRelative()` utility method to set the icon relatively, in a backward-compatible manner.